internal/xpkg/upbound/context.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package upbound contains
+// Package upbound implements authentication for xpkg.upbound.io.
 package upbound
```

Comment:
out of curiousity, what's your thought process for putting logic into `internal/...` instead of under `cmd/crank...`? we had a similar discussion in https://github.com/crossplane/crossplane/pull/4751#discussion_r1359692506

Reply:
Mostly intuition at this point, but my thinking is I'd like to keep `cmd/` focused on "top-level" command logic, with anything more complex living under `internal/`. I considered moving most of the `crossplane beta render` implementation to `internal/xrender` and just leaving the `Run` method under `cmd/`. This is similar to what we do with `cmd/crossplane`, which largely imports and plumbs together libraries from `internal/`.